### DISCUSSION TITLE
Updated pmtiles file failing to load

### DISCUSSION THREAD

**Discussion by Robinlovelace** [protomaps/PMTiles#377] [Q&A] [answered]
Dear PMTiles community, we're bamboozled about a recently-created PMTiles dataset that doesn't show.

Here's the old one that works: https://nptscot.blob.core.windows.net/pmtiles/rnet-2023-12-17.pmtiles

Here's the new one that doesn't: https://nptscot.blob.core.windows.net/pmtiles/rnet-2024-03-11.pmtiles

When serving these locally we were hitting a message saying that the magic number wasn't right, similar to #283 

As far as we can see from the viewer at https://protomaps.github.io/PMTiles/ they both work and have the same data. Are we missing something obvious? 

Cc @mvl22 who is working on the frontend web development side of our Network Planning Tool that this work is part of who may have more comments.

**Comment by Robinlovelace** [comment]
https://github.com/nptscot/nptscot.github.io/issues/156

**Comment by mvl22** [comment, marked as answer]
I've found the issue - it's a filtering issue due to a change of fieldnames. Apologies that neither of us had spotted this as the reason.

This discussion can be closed.

**Comment by bdon** [comment]
tippecanoe 2.48 was creating invalid archives `wrong magic number...`, are you using that? if so, please upgrade

**Comment by Robinlovelace** [comment]
> tippecanoe 2.48 was creating invalid archives `wrong magic number...`, are you using that? if so, please upgrade

I don't think so. Could it be that the local tileserver causes the error? Seems to work when on our remote server on Azure but I only see the magic number issue with localhost..
